src/cli/run.js

```diff
@@ -43,29 +44,39 @@ const findGraphqlTagReferences = (root: string): Array<string> => {
         .map((relative) => path.join(root, relative));
 };
 
-const [_, __, configFile, ...cliFiles] = process.argv;
+const [_, __, configFilePath, ...cliFiles] = process.argv;
 
 if (
-    configFile === '-h' ||
-    configFile === '--help' ||
-    configFile === 'help' ||
-    !configFile
+    configFilePath === '-h' ||
+    configFilePath === '--help' ||
+    configFilePath === 'help' ||
+    !configFilePath
 ) {
     console.log(`graphql-flow
 
 Usage: graphql-flow [configFile.json] [filesToCrawl...]`);
     process.exit(1); // eslint-disable-line flowtype-errors/uncovered
 }
 
-const config = loadConfigFile(configFile);
+const makeAbsPath = (maybeRelativePath: string, basePath: string) => {
+    return maybeRelativePath.startsWith('/')
```

Comment:
why not use `path.isAbsolute(maybeRelativePath)` here? you're OS-agnostic everywhere else, but this will break on a Windows server